testdata/modo/bank_direct_psp_response-paymentResponse.json

```diff
@@ -0,0 +1 @@
+{"$schema":"https://json-schema.org/draft/2019-09/schema","type":"object","properties":{"bank_response_reason":{"description":"the response/reason from the acquirer bank","type":"string"},"payment_method":{"type":"string"},"psp_name":{"description":"Name of the payment service provider","type":"string"},"response_time":{"description":"The delta (in seconds) between when Modo sends a request to psp and receives the response","type":"number"},"status_code":{"description":"The status code returned during the processing of the payment","type":"string"},"status_message":{"description":"The status message (human readable) returned during the processing of the payment","type":"string"}},"additionalProperties":false}
```

Comment:
I'm only a little sad these are minified. :(
If it means I don't have to read them it makes me happier.